### ISSUE TITLE
Restrict edition in a single registry from a table

### ISSUE THREAD

**Issue by Isofff** [volkanceylan/Serenity#1041]
Is there a way to restrict a single registry's edition in a table, according to one of its fields?

For example, one of my tables, "Parameters", has one boolean field, "Editable", that indicates if a particular registry can be edited in runtime or not. Its value cannot be directly changed, so I want to use it to restrict access to the EditLink of that particular registry if false.

What would be the best way to do this?

**Comment by volkanceylan** [comment]
Write a custom formatter, see basic samples for custom links

**Comment by Isofff** [comment]
Thanks, a custom formatter worked.

**Comment by Isofff** [comment]
I followed the examples in CustomLinksInGrid.ts and with a custom formatter I was able to restrict the edition of some registries in my table.

However, after that, my grid doesn't refresh automatically after the update/delete of a valid registry.
The "deleted" row still appears in the grid, and is necessary to click the "refresh" button manually in order to refresh the grid.

I noticed that the same behavior appears in Custom Links In Grid example,  so I reopened this issue seeking for advice in how to refresh the grid automatically after the implementation of a custom formatter.

Thanks in advance.
